### PR TITLE
FEAT: Import nastran improvements

### PR DESCRIPTION
**Pull request by maxcapodi78**
Added Support for CHEXA, PSHELL and MPC in import_nastran method
Pshell are used for Material assignment
MPC for Port assignment
The method is now also returning the dictionary of the objects so user can manipulate and grab information useful to setup and material assignment

### CODE REVIEW

**Review by Samuelopez-ansys — DISMISSED** [review]

File: src/ansys/aedt/core/modules/solve_setup.py [hunk 1]
```diff
@@ -80,6 +80,17 @@ def default_intrinsics(self):
...
+            try:
+                intr["Freq"] = (
+                    self._app.modeler.edb.setups[self.name]
+                    .adaptive_settings.adaptive_frequency_data_list[0]
+                    .adaptive_frequency
+                )
+                intr["Phase"] = "0deg"
+                return intr
+            except Exception:
+                pass
```
Comment (SMoraisAnsys) [inline src/ansys/aedt/core/modules/solve_setup.py:93]:
```suggestion
                settings.logger.debug("Failed to retrieve adaptive frequency.")
```

File: src/ansys/aedt/core/visualization/advanced/misc.py [hunk 2]
```diff
@@ -331,6 +331,14 @@ def get_point(ll, start, length):
...
                 num_points = 5
                 obj_type = "Solids"
+            elif line_type in ["CHEXA", "CHEXA*"]:
+                num_points = 8
+                obj_type = "Solids"
+            elif line_type in ["PSHELL", "PSHELL*"]:
+                num_points = 1
+                obj_type = "Shells"
+            elif line_type in ["MPC", "MPC*"]:
+                num_points = 16
```
Comment (SMoraisAnsys) [inline src/ansys/aedt/core/visualization/advanced/misc.py:341]: Shouldn't you affect `obj_type` here also ?

**Review by SMoraisAnsys — COMMENTED** [review]
Thanks for the changes, I like the idea of returning the dictionary. I left minor comments.
Btw, could you improve the code coverage of the PR ?

File: src/ansys/aedt/core/visualization/advanced/misc.py (hunk 2 above)
Comment (maxcapodi78) [reply src/ansys/aedt/core/visualization/advanced/misc.py:341]: nope because I don't that info for MCP

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**